README.md

```diff
@@ -107,7 +97,7 @@ To install this project, run
 

```

Comment:
![image](https://github.com/Bestbynature/Morse-Code-Interpreter/assets/93039359/cba222f1-9792-44d6-b9cb-caf0b0c263b9)

- [ ] It would be nice if you fix the highlighted links in the image above because these links are broken 😨 Kindly fix them or remove them **(if you don't have any section for these links)** to make your readme descriptive and professional 👍